src/features/form/validations.js

```diff
@@ -20,6 +20,42 @@ const dateToInt = dateStr => new Date(dateStr).getTime();
 const maxDateValue = maxDate => value =>
   value && (dateToInt(value)) > dateToInt(maxDate) ? `Date should be earlier or equal to today's date` : undefined;
 
+const getNormalRangeFromConcept = concept => {
+  const {
+    hiAbsolute,	
+    lowAbsolute,
+  } = concept;
+  let hiAbsoluteRange, lowAbsoluteRange;
+
+  if (hiAbsolute || lowAbsolute) {
+    hiAbsoluteRange = hiAbsolute ? maxValue(hiAbsolute) : undefined;
+    lowAbsoluteRange = hiAbsolute ? minValue(lowAbsolute) : undefined;
+    return [hiAbsoluteRange, lowAbsoluteRange].filter(Boolean);
```

Comment:
@jomadoye what does filter(Boolean) do?  Filtered out undefined?

Reply:
yes it filters out any falsy value